src/r_main.c

```diff
@@ -242,14 +242,14 @@ static void FlipCam2_OnChange(void)
 }
 
 //
-// R_PointOnSide
+// R_OldPointOnSide
 // Traverse BSP (sub) tree,
 // check point against partition plane.
 // Returns side 0 (front) or 1 (back).
 //
 // killough 5/2/98: reformatted
 //
-INT32 R_PointOnSide(fixed_t x, fixed_t y, node_t *node)
+PUREFUNC INT32 R_OldPointOnSide(fixed_t x, fixed_t y, const node_t *restrict node)
```

Comment:
@chromaticpipe FYI, I already tested this and it doesn't change the resulting assembly code. No harm in adding it but nothing gained by it either.